pkg/deploy/generator/resources.go

```diff
@@ -1513,6 +1513,27 @@ func (g *generator) rbac() []*arm.Resource {
 		},
 	}
 }
+func (g *generator) rbac_prod() []*arm.Resource {
+	return []*arm.Resource{
+		{
+			Resource: &mgmtauthorization.RoleAssignment{
+				Name: to.StringPtr("[concat(parameters('databaseAccountName'), '/Microsoft.Authorization/', guid(resourceId('Microsoft.DocumentDB/databaseAccounts', parameters('databaseAccountName')), '970792b5-7720-4bf5-a335-f15e97c7e25a' , 'Billing / DocumentDB Account Contributor'))]"),
```

Comment:
What is `970792b5-7720-4bf5-a335-f15e97c7e25a`?  Can we make this a constant or abstract it away?

Reply:
moved to constant